packages/legend-extension-dsl-serializer/src/models/metamodels/pure/model/packageableElements/schemaSet/Schema.ts

```diff
@@ -24,11 +24,14 @@ export class Schema implements Hashable {
   location?: string | undefined;
   content: string;
 
-  constructor(id: string, location: string, content: string) {
+  constructor(content: string, id?: string, location?: string) {
```

Comment:
Let's avoid this, passing optional parameters to constructor like this is less than ideal, also remember what we said last time about using a `dumb` constructor, let's get rid of the parameters and populate them after instantiation